mkosi/__init__.py

```diff
@@ -6739,49 +6743,48 @@ def find_qemu_binary() -> str:
 
 
 def find_qemu_firmware() -> Tuple[str, bool]:
-    # UEFI firmware blobs are found in a variety of locations,
-    # depending on distribution and package.
-    FIRMWARE_LOCATIONS = []
-
-    if platform.machine() == "x86_64":
-        FIRMWARE_LOCATIONS.append("/usr/share/ovmf/x64/OVMF_CODE.secboot.fd")
-    elif platform.machine() == "i386":
-        FIRMWARE_LOCATIONS.append("/usr/share/edk2/ovmf-ia32/OVMF_CODE.secboot.fd")
-
-    FIRMWARE_LOCATIONS.append("/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd")
-    FIRMWARE_LOCATIONS.append("/usr/share/qemu/OVMF_CODE.secboot.fd")
-    FIRMWARE_LOCATIONS.append("/usr/share/ovmf/OVMF.secboot.fd")
+    FIRMWARE_LOCATIONS = [
+        # UEFI firmware blobs are found in a variety of locations,
+        # depending on distribution and package.
+        {
+            "x86_64": "/usr/share/ovmf/x64/OVMF_CODE.secboot.fd",
+            "i386": "/usr/share/edk2/ovmf-ia32/OVMF_CODE.secboot.fd",
+        }.get(platform.machine()),
+        "/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd",
+        "/usr/share/qemu/OVMF_CODE.secboot.fd",
+        "/usr/share/ovmf/OVMF.secboot.fd",
+    ]
 
     for firmware in FIRMWARE_LOCATIONS:
-        if os.path.exists(firmware):
+        if firmware and os.path.exists(firmware):
             return firmware, True
 
     warn(
-        """\
-        Couldn't find OVMF firmware blob with secure boot support,
-        falling back to OVMF firmware blobs without secure boot support.
-        """
+        "Couldn't find OVMF firmware blob with secure boot support, "
```

Comment:
This removes a linebreak from the message.

Reply:
Yeah, on purpose — it's generally better to have error messages on one line. If on a terminal, it'll be wrapped automatically.